site/docs/src/json/tenants/request.json

```diff
@@ -3,15 +3,26 @@
     "data": {
       "description": "No more secrets, Marty."
     },
+    "captchaConfiguration": {
+      "captchaMethod": "GoogleRecaptchaV3",
+      "enabled": true,
+      "secretKey": "6LdYWpMbAAAAAKhcksgk70us00012r66r96tt7rp",
```

Comment:
Are these live? Would suggest changing to `YOURKEY` or something similar.

And rotate these now :)

Reply:
Again fake, but I can stop using randomly generated IDs, Keys, etc.  Is there a standard we use for `YOURKEY`, `YOURID`?

Reply:
Nah, if they are fake, that's fine. We don't have a standard on this, other than making sure no one commits real keys :)